Resources/MobileVLCKit/patches/0006-contrib-gcrypt-work-around-a-libtool-limitation.patch

```diff
@@ -31,27 +31,26 @@ index 0000000000..df97ffb488
 +--- libgcrypt-broken/cipher/Makefile.am	2013-11-06 23:05:24.000000000 +0100
 ++++ libgcrypt/cipher/Makefile.am	2013-11-06 23:21:25.000000000 +0100
 +@@ -19,6 +19,8 @@
-+ 
++
 + # Process this file with automake to produce Makefile.in
-+ 
++
 ++LIBTOOL=@LIBTOOL@ --tag=CC
 ++
 + EXTRA_DIST = Manifest
-+ 
++
 + # Need to include ../src in addition to top_srcdir because gcrypt.h is
 +diff -ru libgcrypt-broken/mpi/Makefile.am libgcrypt/mpi/Makefile.am
 +--- libgcrypt-broken/mpi/Makefile.am	2013-11-06 23:05:24.000000000 +0100
 ++++ libgcrypt/mpi/Makefile.am	2013-11-06 23:22:04.000000000 +0100
 +@@ -23,6 +23,9 @@
-+ 
++
 + # Need to include ../src in addition to top_srcdir because gcrypt.h is
 + # a built header.
 ++
 ++LIBTOOL=@LIBTOOL@ --tag=CC
 ++
 + AM_CPPFLAGS = -I../src -I$(top_srcdir)/src
 + AM_CFLAGS = $(GPG_ERROR_CFLAGS)
-+ 
--- 
++
+--
 2.19.1
-
```

Comment:
You misunderstood me. The point I wanted to bring across is about merging, not about the actual content.
Until very recently (and it can still be the case with current public releases), the "patch" utility on macOS is often failing if spaces in the original file are not reflected in the diff. This problem does not occur on Linux, but was biting quite often on the Mac. Hence, while git still complains, it is probably best to keep the spaces. Please try this thoroughly before you merge.